### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by dirknbr** [microprediction/timemachines#14] [Q&A] [answered]
I am using this for the first time. I wanted to test one of the algos and so ran this. But I get an error, what do I need to put into `s`?

```
from timemachines.skaters.simple.thinking import thinking_slow_and_fast
import numpy as np

y = np.arange(100) + np.random.normal(size=100)

res = thinking_slow_and_fast(y, s=None, k=10)
```

**Comment by microprediction** [comment, marked as answer]
Hi Dirk,
   You feed the skater one data point at a time, and the first time send an empty dictionary. 

        y_hat, y_std, s = thinking_slow_and_fast(y[0], s={}, k=10)
        y_hat, y_std, s = thinking_slow_and_fast(y[1], s=s, k=10)
        y_hat, y_std, s = thinking_slow_and_fast(y[2], s=s, k=10)


You can also use the prior or posterior helper functions in [skating.py](https://github.com/microprediction/timemachines/blob/main/timemachines/skating.py) to avoid a loop

Peter
